app/views/estimates/show.html.slim

```diff
@@ -42,6 +42,9 @@ p.govuk-body = t(".next_steps_paragraph_2")
 
 h2.govuk-heading-m = t(".how_calculated")
 = govuk_accordion do |accordion|
+  - accordion.section(heading_text: t(".limits_calculation"))
```

Comment:
Do we want this to appear when the page is printed as well? If so, we'll want to add this to print.html.slim as well.

Reply:
fixed